src/registrar/admin.py

```diff
@@ -18,3 +37,5 @@ class MyUserAdmin(UserAdmin):
 
 
 admin.site.register(User, MyUserAdmin)
+admin.site.register(DomainApplication, AuditedAdmin)
```

Comment:
**question (non-blocking):** Do we need/want tests on our custom admin views? Is it easy enough to make a `TestCase` that makes an admin user, uses `force_login`, and loads one or more of these admin pages?

Reply:
I think that's a great idea, but I'm just going to make a note on #14 about it for the moment.